pkg/schema/tracker.go

```diff
@@ -54,11 +56,20 @@ var (
 
 // Tracker is used to track schema locally.
 type Tracker struct {
-	store kv.Storage
-	dom   *domain.Domain
-	se    session.Session
+	store           kv.Storage
+	dom             *domain.Domain
+	se              session.Session
+	downstreamTrack map[string]*ast.CreateTableStmt // downstream tracker tableid -> createTableStmt
```

Comment:
Why not just store a `*module.TableInfo` here

Reply:
Yes, use `*model.TableInfo` will be more convenient.
I have tried to get  `*model.TableInfo`  from `*ast.CreateTableStmt` which was parsed by `*parser.Parser`, but the value about  `*model.TableInfo`  is nil.

Reply:
There is a function `ddl. MockTableInfo ` that can do this. See:
https://github.com/pingcap/tidb/blob/master/ddl/mock.go#L156-L171